pages/_app.js

```diff
@@ -1,9 +1,36 @@
 import "../styles/globals.css";
 import Layout from "../components/Layout";
+import useLocalStorageState from "../components/Hooks/useLocalStorageState";
+
 function MyApp({ Component, pageProps }) {
+  // Custom hooks setting user user details - here in the _app.js so it can be accessed acrosss all componenets!
+  const [learnerName, setLearnerName] = useLocalStorageState(
+    "learner-name",
+    null
+  ) || [null, null];
+  const [font, setFont] = useLocalStorageState("font", null) || [null, null];
+  const [size, setSize] = useLocalStorageState(
+    "font-size",
+    null
+  ) || [null, null];
+  const [background, setBackground] = useLocalStorageState(
+    "background",
+    null
+  ) || [null, null];
+
   return (
     <Layout>
-      <Component {...pageProps} />
+      <Component
```

Comment:
Great work on the specific route once learner navigates, the props become active and set these components to become active ✨